apps/i18n/common/en_us.json

```diff
@@ -648,7 +651,7 @@
   "projectTypeBounce": "Bounce",
   "projectTypeInfinity": "Infinity",
   "projectTypeIceage": "Ice Age",
-  "projectTypeGumball": "The Amazing World of Gumball",
+  "projectTypeGumball": "Gumball",
```

Comment:
is this used anywhere else on the site? When not in this form, it's important we use the full show name.

Reply:
Actually @poorvasingal did we get permission to just use the name "Gumball"? They were specific about it elsewhere on the site.

Reply:
It's not used anywhere else that I can see:
<img width="902" alt="screen shot 2017-11-06 at 3 07 38 pm" src="https://user-images.githubusercontent.com/8001765/32468728-552bde22-c304-11e7-92c5-568e44648391.png">